client/lib/resurrected-users/README.md

```diff
@@ -0,0 +1,49 @@
+# Resurrected Free Users Experiment Foundation
+
+This module contains the reusable primitives that power the `calypso_resurrected_users_welcome_back_modal_202511`
+experiment:
+
+- `useResurrectedFreeUserEligibility` – Computes experiment eligibility (180 day inactivity threshold,
+  account-level purchase checks, feature-flag gates, and ExPlat assignment).
+- Analytics helpers – shared constants for Tracks event names and variation keys.
+- `ResurrectedWelcomeModalGate` – A lightweight modal placeholder that renders the correct variant slot and
+  emits impression/CTA events when the user is eligible.
+
+## Flow Overview
+
+1. `TrackResurrections` now logs both the legacy 373 day event and a new `calypso_user_resurrected_6m` event so we can
```

Comment:
Nice, adding a separate event for the shorter period we'll use in the experiment is a good idea.